docs/next-steps/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Next Steps
+description: A guide to making a living in the Stacks world
+sidebar_position: 7
+---
+# Furthering Your Stacks Skills and Career
+The Stacks ecosystem is rapidly growing, and there are currently multiple ways for interested developers to monetize their hard work and make a living building the next generation of the Internet.
+
+## Awesome Stacks
+Awesome Stacks is a community-curated collection of resources for learning and building on Stacks. It has tons of helpful developer tools, companies, and learning resources to help you further your Stacks knowledge.
+
+[View Awesome Stacks Repo](https://github.com/friedger/awesome-stacks-chain)
```

Comment:
i would remove `View` here, but that's an opinionated change. I don't feel it adds to the description